.github/workflows/builder_image_tests.yml

```diff
@@ -18,38 +20,102 @@ env:
-          - quarkus-version: '3.6.0'
-            mandrel-builder-image: 'jdk-21'
-          - quarkus-version: '3.8.4'
-            mandrel-builder-image: 'jdk-21'
-          - quarkus-version: '3.9.5'
-            mandrel-builder-image: 'jdk-21'
-          - quarkus-version: '3.9.5'
-            mandrel-builder-image: 'jdk-22'
-          - quarkus-version: '3.10.0'
-            mandrel-builder-image: 'jdk-22'
+            mandrel-builder-version: 'jdk-17'
+            mandrel-builder-base: 'ubi9'
+            os: 'ubuntu-24.04'
+
+          - quarkus-version: '3.8.6.1'
+            mandrel-builder-version: 'jdk-21'
+            mandrel-builder-base: 'ubi'
+            os: 'ubuntu-22.04'
+
+          - quarkus-version: '3.15.4'
+            mandrel-builder-version: 'jdk-21'
+            mandrel-builder-base: 'ubi'
+            os: 'ubuntu-22.04'
+
+          - quarkus-version: '3.20.0'
+            mandrel-builder-version: 'jdk-21'
+            mandrel-builder-base: 'ubi'
+            os: 'ubuntu-22.04'
+
+          - quarkus-version: '3.21.0'
+            mandrel-builder-version: 'jdk-21'
+            mandrel-builder-base: 'ubi'
+            os: 'ubuntu-22.04'
+          - quarkus-version: '3.21.0'
+            mandrel-builder-version: 'jdk-21'
+            mandrel-builder-base: 'ubi9'
+            os: 'ubuntu-24.04'
+          - quarkus-version: '3.21.0'
+            mandrel-builder-version: 'jdk-21'
+            mandrel-builder-base: 'ubi'
+            os: 'ubuntu-22.04-arm'
+          - quarkus-version: '3.21.0'
+            mandrel-builder-version: 'jdk-21'
+            mandrel-builder-base: 'ubi9'
+            os: 'ubuntu-24.04-arm'
+
+          - quarkus-version: '3.8.6.1'
+            mandrel-builder-version: 'jdk-22'
+            mandrel-builder-base: 'ubi'
+            os: 'ubuntu-22.04'
+          - quarkus-version: '3.8.6.1'
+            mandrel-builder-version: 'jdk-22'
+            mandrel-builder-base: 'ubi9'
+            os: 'ubuntu-24.04'
+
+          - quarkus-version: '3.21.0'
+            mandrel-builder-version: 'jdk-23'
+            mandrel-builder-base: 'ubi'
+            os: 'ubuntu-22.04'
+          - quarkus-version: '3.21.0'
+            mandrel-builder-version: 'jdk-23'
+            mandrel-builder-base: 'ubi9'
+            os: 'ubuntu-24.04'
+
+         # - quarkus-version: '3.21.0'
+         #   mandrel-builder-version: 'jdk-24'
+         #   mandrel-builder-base: 'ubi'
+         #   os: 'ubuntu-22.04'
+         # - quarkus-version: '3.21.0'
+         #   mandrel-builder-version: 'jdk-24'
+         #   mandrel-builder-base: 'ubi9'
+         #   os: 'ubuntu-24.04'
+         #- quarkus-version: '3.21.0'
+         #  mandrel-builder-version: 'jdk-24'
+         #  mandrel-builder-base: 'ubi'
+         #  os: 'ubuntu-22.04-arm'
+         #- quarkus-version: '3.21.0'
+         #  mandrel-builder-version: 'jdk-24'
+         #  mandrel-builder-base: 'ubi9'
+         #  os: 'ubuntu-24.04-arm'
```

Comment:
Could we just remove this, please?

Reply:
I will enable it and try for https://mvnrepository.com/artifact/io.quarkus/quarkus-core/3.22.0.CR1 which should have https://github.com/quarkusio/quarkus/pull/46731

Reply:
I tried to enable JDK 24 with Quarkus 3.22.0.CR1 an I disabled AWT realted things on aarch64for JDK24.

Quarkus 3.22.0.CR1  keeps failing **all**  quarkus-mp-orm-dbs-awt app's test on a classloading issue.
It seems to me to be related to https://github.com/quarkusio/quarkus/pull/34681#issuecomment-2816730589

Reply:
Resolved by surefire update.